.github/workflows/macos.yml

```diff
@@ -51,6 +51,7 @@ jobs:
       run: |
         cmake stratagus -B stratagus/build \
         -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_FIND_FRAMEWORK=LAST \
```

Comment:
No need for "Build Stargus"? as you do for wargus/war1gus.

Reply:
This flag doesn't work with Meson.

Meson is building with `1.6.45`, which is older than the latest `1.6.47` but it's new enough not to cause issues for now.